src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,181 @@
+          </span>
+        </p>
+      </div>
+
+      <div className="panel-block">
+        <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
+          <div className="level-left">
+            {CENTURIES.map(century => (
+              <SearchLink
+                params={{ centuries: toggleCenturies(century) }}
+                key={century}
+                data-cy="century"
+                className={classNames('button mr-1', {
+                  'is-info': centuries.includes(century.toString()),
+                })}
+              >
+                {century}
+              </SearchLink>
+            ))}
+          </div>
+          {/* <div className="panel-block">
+        <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
+          <div className="level-left">
+            {CENTURIES.map(century => (
+              <a
+                key={century}
+                data-cy="century"
+                className={classNames('button mr-1', {
+                  'is-info': centuries.includes(century.toString()),
+                })}
+                onClick={() => toggleCenturies(century)}
+              >
+                {century}
+              </a>
+            ))}
+          </div> */}
+
+          <div className="level-right ml-4">
+            <a
+              data-cy="centuryALL"
+              className="button is-success is-outlined"
+              href="#/people"
+            >
+              All
+            </a>
+          </div>
+        </div>
+      </div>
+
+      <div className="panel-block">
+        <a
+          className="button is-link is-outlined is-fullwidth"
+          onClick={clearCenturies}
+          href="#/people"
+        >
+          Reset all filters
+        </a>
+      </div>
+    </nav>
+  );
+};
+
+{
+  /* <div className="panel-block">
+        <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
+          <div className="level-left">
+            <a
+              data-cy="century"
+              className="button mr-1"
+              href="#/people?centuries=16"
+              onClick={handleCenturiesChange}
+            >
+              16
+            </a>
+
+            <a
+              data-cy="century"
+              className="button mr-1 is-info"
+              href="#/people?centuries=17"
+            >
+              17
+            </a>
+
+            <a
+              data-cy="century"
+              className="button mr-1 is-info"
+              href="#/people?centuries=18"
+            >
+              18
+            </a>
+
+            <a
+              data-cy="century"
+              className="button mr-1 is-info"
+              href="#/people?centuries=19"
+            >
+              19
+            </a>
+
+            <a
+              data-cy="century"
+              className="button mr-1"
+              href="#/people?centuries=20"
+            >
+              20
+            </a>
+          </div> */
+}
```

Comment:
on GH code should be clean and ready for review, remove commented

Reply:
Done